transformers/socket.io/client.js

```diff
@@ -22,15 +22,15 @@ module.exports = function client() {
   var factory = (function factory() {
     if ('undefined' !== typeof io && io.Socket) return io;
 
-    try { return Primus.require('socket.io-client'); }
+    try { return Primus.require('primus-socket.io-client'); }
```

Comment:
It might make sense to fallback to `socket.io-client` here so opting-in for these fixes is optional. As the API stays the same.